taipy/core/common/_utils.py

```diff
@@ -30,11 +31,25 @@ def _get_fct_name(f) -> Optional[str]:
 
 
 def _fct_to_dict(obj):
-    fct_name = _get_fct_name(obj)
+    params = []
+    callback = obj
```

Comment:
is the variable callback useful? can't we use obj directly?

Reply:
Obj in this case can be either a direct function or it can be a tuple(fct, params), if it is the tuple we need to check and replace the callback variable.